mobile_app_connector/security/ir.model.access.csv

```diff
@@ -38,3 +38,4 @@ admin_access_mobile_app_tile_subtype,admin access_mobile_app_tile_subtype,model_
 admin_access_mobile_app_writing,full access_mobile_app_writing,model_mobile_app_writing,sales_team.group_sale_manager,1,1,1,1
 user_access_mobile_app_feedback,User access app feedback,model_mobile_app_feedback,base.group_portal,1,1,1,0
 admin_access_mobile_app_feedback,Admin access app feedback,model_mobile_app_feedback,base.group_system,1,1,1,1
+access_frequently_asked_questions_default,access_frequently_asked_questions_default,model_frequently_asked_questions,base.group_user,1,0,0,0
```

Comment:
Access rules are not necessary for abstract models.

Reply:
Done.